csv.md

```diff
@@ -0,0 +1,124 @@
+
+#### Dimension
+
+Dimension columns (otherwise known as factors) are used to identify the observation through a combination of concepts. Where each dimension in a CSV is filtered to a specific value there should only be one observation. In relational databases terminology all dimensions form to be a unique composite key. Some examples of dimensions are:
+
+- `government_year` with one value being `2019-2020` (i.e. the period of `April 2019 to March 2020`)
+- `geography_code` with one value being `E09000001` (i.e. the nation of `England`)
+- `sic_2007` with one value being `01.11` (i.e. the concept `Growing of cereals (except rice), leguminous crops and oil seeds`)
+
+To improve human readability and to ensure machine readability, ensure that codes in separate columns from the related human readable labels or descriptions. While the labels and additional information can be useful for humans, mixing them in the same column with codes makes users have to clean the data. It is also important to ensure that related columns are grouped together when sorted alphabetically, and to make it easier to find the human readable values.
+
+A quick way to check if a column only contains related data and not unique identifiable is if you filter on a column and for every value you select another column will only ever have the same value for that row it is likely to help human readability.
+
+For example the three columns prefixed with `geo_` are related in the table below, filtering on any of the three would only ever result in one value for the other two columns. The geo_code column is a unique identifier for the geography.
+
+| geo_code  | geo_name          | geo_type               | value | ... |
+|-----------|-------------------|------------------------|-------|-----|
+| E08000006 | Salford           | Metropolitan Districts | 42    | ... |
+| E92000001 | England           | Country                | 1337  | ... |
+| K04000001 | England and Wales | England and Wales      |       | ... |
+
+**Note:** Dimension columns must contain values for every row in the CSV file and not be blank (i.e. they must be dense)
+
+#### Attributes
+
+Attribute columns are used to describe the observation. They can be used to describe the observation as a whole. Most commonly the attribute columns are used to describe the absence or quality of an observation, these are commonly called "observation status" columns.
+
+##### Observation status columns
+
+When creating observation status columns a naming convetion helps users understand how they relate to the observation to the qualification. In this case for a given column name containing observations, the observation status column column should have the same name as the observation column plus `_obs_status` as a suffix. For example an observation column called `gva` should have a corresponding observation status called `gva_obs_status` which should be located to the right of the observation column.
```

Comment:
Type at 'naming convetion helps users...'
Is this meant to say 'helps users understand how they relate the observation to the qualification'?